@xen-orchestra/fs/src/abstract.js

```diff
@@ -362,18 +366,27 @@ export default class RemoteHandlerAbstract {
   }
 
   async test(): Promise<Object> {
+    const SIZE = 1024 * 1024 * 10
     const testFileName = normalizePath(`${Date.now()}.test`)
-    const data = await fromCallback(cb => randomBytes(1024 * 1024, cb))
+    const data = await fromCallback(cb => randomBytes(SIZE, cb))
     let step = 'write'
     try {
+      const startWrite = process.hrtime()
       await this._outputFile(testFileName, data, { flags: 'wx' })
+      const endWrite = process.hrtime(startWrite)
```

Comment:
Contrary to `startWrite`, this is not an moment in time, it's a duration, I think we should rename this.